packages/flutter/lib/src/cupertino/desktop_text_selection_toolbar.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+// Minimal padding from all edges of the selection toolbar to all edges of the
+// screen.
+const double _kToolbarScreenPadding = 8.0;
+
+// These values were measured from a screenshot of TextEdit on MacOS 10.15.7 on
+// a Macbook Pro.
+const double _kToolbarWidth = 222.0;
```

Comment:
Hello Justin !

It seems that the width is not fixed and depends on the widest menu item according to [Apple documentation](https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-anatomy/). 

![image](https://user-images.githubusercontent.com/39876339/104600631-8b1e9180-5679-11eb-8f32-5fa0b14b67c8.png)

Reply:
Good point, thanks!  I'll make that a minimum width.

Reply:
This is not as easy as I thought.  I think it might need a custom layout widget.  The parent needs to size itself to the biggest intrinsic width of its children, and the children then need to all match that.

I've added this to the "Known limitations" above since this PR is somewhat rushed.  Given more time this is definitely doable, though.

Reply:
Similar to https://github.com/flutter/flutter/issues/72023.

Reply:
Okok perfect!